bin/_main

```diff
@@ -111,11 +126,26 @@ elif [ "${1}" == "run" ]; then
     else
         $podman_pull $CRUCIBLE_CONTAINER_IMAGE
     fi
-    mkdir -p "$base_run_dir"
+    mkdir -pv "$base_run_dir/config"
     benchmark_subproj_dir="${CRUCIBLE_HOME}"/subprojects/benchmarks/$benchmark
     rs_dir="${CRUCIBLE_HOME}"/subprojects/core/rickshaw
 
-    if [ ! -e ${bench_params} ]; then
+    if [ "${use_mv_params}" == "1" ]; then
+        if [ ! -e "${mv_params}" ]; then
+            echo "The multi-value params file you specified with --mv-params (${mv_params}) does not exist!"
+            exit 1
+        else
+            mv_params_run_dir=${base_run_dir}/config/mv-params.json
+            cp -v ${mv_params} ${mv_params_run_dir}
+            bench_params_run_dir=${base_run_dir}/config/bench-params.json
+
+            echo "Generating --bench-params from --mv-params..."
+            echo "$podman_run -i --name crucible-multiplex "${container_common_args[@]}" $CRUCIBLE_CONTAINER_IMAGE /opt/crucible/subprojects/core/multiplex/multiplex.py --input ${mv_params_run_dir} > ${bench_params_run_dir}"
+            $podman_run -i --name crucible-multiplex "${container_common_args[@]}" $CRUCIBLE_CONTAINER_IMAGE /opt/crucible/subprojects/core/multiplex/multiplex.py --input ${mv_params_run_dir} > ${bench_params_run_dir}
```

Comment:
Did you rewrite mulitplex in python?

Reply:
Yes.  I guess I now know why you never reviewed my PR...